extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -73,7 +73,7 @@ const PRE_RENDER_REGEX = new RegExp(`${COUNTRY_PREFIX}(\\w+)`);
 const GEO_ID = 'ampGeo';
 const SERVICE_TAG = 'geo';
 const API_TIMEOUT = 60; // Seconds
-const GEO_HOTPATCH_STR_REGEX = /^(\w{2})?(\s(\w{2}-\w{2}))?\s*/;
+const GEO_HOTPATCH_STR_REGEX = /^(?:(\w{2})(?:\s(\w{2}-\w{2}))?)?\s*/;
```

Comment:
should have asked Justin for help on regex rather than spending an entire afternoon on this lol : )

Reply:
FWIW, ISO 3166-1 expects `/[a-z]{2}/i` and ISO 3166-2 expects `/[a-z]{2}-[a-z0-9]{2}/i`, so these could be tightened a bit instead of using `\w`.